test/conftest.py

```diff
@@ -147,33 +174,49 @@ def test_workspaces(
     )
 
 
-@pytest.fixture(scope="session", autouse=True)
-def ws_id(test_workspaces: dict[str, Any]) -> int:
-    return test_workspaces["ws_id"]
+@pytest.fixture(scope="session")
+def default_ws_id(test_workspaces: dict[str, Any]) -> int:
```

Comment:
rename the fixtures for the default workspace to more accurately reflect their identities.